src/com/lftechnology/java/training/alina/jdbc/service/UserService.java

```diff
@@ -0,0 +1,339 @@
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class UserService {
+
+    private static final Logger LOGGER = Logger.getLogger(UserService.class.getName());
+
+    /**
+     * Gets login info
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @return user {@link User} user info
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     * @throws SQLException
+     */
+    public static User setLoginInfo(Scanner scanner, String actionType) throws SQLException {
+        User user = new User();
+        boolean userExist = false;
+        if (actionType == Constants.USER_LOGIN) {
+            user.setUsername(UtilityService.getInputData(scanner, "Enter Username : "));
+        } else {
+            do {
+                String username = UtilityService.getInputData(scanner, "Enter Username : ");
+                userExist = LoginController.checkExistUsername(userExist, username);
+                if (!userExist) {
+                    user.setUsername(username);
+                }
+            } while (userExist);
+        }
+        user.setPassword(UtilityService.getInputData(scanner, "Enter Password : "));
+        user.setTerminated(true);
+        user.setCreatedAt(DateTimeService.getCurrentTimeStamp());
+        user.setModifiedAt(DateTimeService.getCurrentTimeStamp());
+        return user;
+    }
+
+    /**
+     * Used to get Role of an employee
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param result
+     *            {@link ResultSet}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    public void getEmployeeRole(Scanner scanner, Employee employee) throws SQLException {
+        if (employee.getRole().equals(EmployeeRole.ADMIN.role)) {
+            getAdminRole(scanner, employee);
+        } else {
+            getNormalUserRole(scanner, employee);
+        }
+    }
+
+    /**
+     * Gets Admin role functions
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param result
+     *            {@link ResultSet}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    private void getAdminRole(Scanner scanner, Employee employee) throws SQLException {
+        char choice = ' ';
+        while (choice != Constants.ADMIN_EXIT) {
+            AdminView.displayAdminRoleMenu(employee);
+            choice = UtilityService.getSelectedMenu(scanner, "Select an option (a-f) : ");
+            getAdminRoleOptions(scanner, choice);
+        }
+    }
+
+    /**
+     * Gets option menu for Admin Role
+     * 
+     * @param scanner
+     *            {@link Scanner}
+     * @param option
+     *            {@link Character}
+     * @throws SQLException
+     * @author Alina Shakya <alinashakya@lftechnology.com>
+     */
+    private void getAdminRoleOptions(Scanner scanner, char option) throws SQLException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getAdminRoleOptions" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)